terragrunt/aws/ecr/iam_oidc.tf

```
@@ -98,4 +98,48 @@ data "aws_iam_policy_document" "ecr_image_manage" {
     ]
     resources = ["*"]
   }
-}
\ No newline at end of file
+}
+
+data "aws_iam_policy_document" "pr_review_env_policy_document" {
+  statement {
+    effect = "Allow"
+
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+
+    resources = [
+      "arn:aws:logs:${var.region}:${var.account_id}:*"
+    ]
+  }
+}
+
+resource "aws_iam_role_policy_attachment" "pr-review-env-lambda" {
+  role       = aws_iam_role.pr-review-env-lambda.name
+  policy_arn = aws_iam_policy.pr_review_env.arn
```

Just a minor tweak - this needs to be:
```suggestion
  policy_arn = aws_iam_policy.pr-review-env-lambda.arn
```